<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="display_2">
        <property name="text">
         <string>display</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="display">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Sans Serif&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QPushButton" name="put_data_2">
        <property name="text">
         <string>put_data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="put_data"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="c10to2">
        <property name="text">
         <string>c10to2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="c10to16">
        <property name="text">
         <string>c10to16</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="c2to10">
        <property name="text">
         <string>c2to10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="c16to10">
        <property name="text">
         <string>c16to10</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="sin">
        <property name="text">
         <string>sin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cos">
        <property name="text">
         <string>cos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tan">
        <property name="text">
         <string>tan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="log">
        <property name="text">
         <string>log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="powerfunction">
        <property name="text">
         <string>x^a</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="gy">
        <property name="text">
         <string>gy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gb">
        <property name="text">
         <string>gb</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="compare">
        <property name="text">
         <string>compare</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="blank">
        <property name="text">
         <string>blank</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="jiecheng">
        <property name="text">
         <string>n!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="qf">
        <property name="text">
         <string>y=ax^2+bx+c</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="expf">
        <property name="text">
         <string>y=a^x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logfunction">
        <property name="text">
         <string>y=log(a)x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="firstof">
        <property name="text">
         <string>y=kx+b</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fan">
        <property name="text">
         <string>y=k/x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_AC">
        <property name="text">
         <string>AC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_CE">
        <property name="text">
         <string>CE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="button_4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_add">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_minus">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="button_7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_ride">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_divide">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="button_point">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_left">
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_right">
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_equal">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QPushButton" name="A">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="B">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="C">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D">
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="E">
        <property name="text">
         <string>E</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="F">
        <property name="text">
         <string>F</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menucalculator">
    <property name="title">
     <string>calculator</string>
    </property>
   </widget>
   <addaction name="menucalculator"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
